index.html

```diff
@@ -165,20 +272,120 @@ <h4>Skills</h4>
             <li>Gitlab</li>
             <li>Terminal</li>
         </ul>
+
+        <!-- For media css -->
+
     </section>
+    <section class="content">
+        <h3 class="for-med-only">
+            I'm always interested in hearing about new projects, so if you'd 
+            like to chat please get in touch.
+        </h3>
+
+    </section>
+    <section class="mediacss-about-me">
+        <h2 class="ab-me">About me</h2>
+        <p class="ab-p">Hello I’m a software developer! I can help you build <br> 
+            a product , feature or website Look through some of my work <br> 
+            and experience! If you like what you see and have a project <br>
+            you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="button3" type="button">Get My Resume</button>
```

Comment:
- [ OPTIONAL ] Kindly fix the get my resume font by removing the `font-family: 'Inter', sans-serif;` because it makes the font of the text bigger.
 Your design
![Screenshot from 2022-07-08 15-33-59](https://user-images.githubusercontent.com/77842237/177993369-39fa0cd7-4870-41bd-ba91-b4402697f056.png)